### PR TITLE
Add updatedAt in ticket card, Resolves issue #12

### LINKED ISSUE #12: Show when was the ticket last updated.
Currently we can see when was the ticket created , we want to see when was it last updated with changes as well.

### PR DESCRIPTION
**Pull request by MadhavDhall**
### Related Issue  
Closes #12 

### Type of Change
Put `x` inside the square bracket to specify what type of change your PR is:  
- [x] New Feature  
- [ ] Bug Fix  
- [ ] Code Refactor  
- [ ] Documentation Update  
- [ ] Other (please specify): 

### Description of Change  
When was ticket last updated time and date is now shown below the ticket creation time along with mention of which time is creation and which one is updation time.

### Implementation Details  
Last update time is fetched from db and shown in home page in each ticket.
### Demo  
Screenshot 2024-12-25 at 11 21 31 PM

### CODE REVIEW

**Comment by MadhavDhall** [comment]
Kindly review the PR @AyushDharDubey @himani-r5 .

File: app/(components)/TicketCard.jsx [hunk 1]
```diff
@@ -20,12 +20,12 @@ const TicketCard = ({ ticket }) => {
...
 
   const createdDateTime = formatTimestamp(ticket.createdAt);
+  const updatedDateTime = formatTimestamp(ticket.updatedAt);
 
   return (
     <div
-      className={`flex flex-col hover:bg-card-hover bg-card rounded-md shadow-lg p-3 m-2 ${
-        ticket.priority > 4 ? 'bg-red-500' : ''
-      }`} 
+      className={`flex flex-col hover:bg-card-hover bg-card rounded-md shadow-lg p-3 m-2 ${ticket.priority > 4 ? 'bg-red-500' : ''
```
Comment (AyushDharDubey) [inline app/(components)/TicketCard.jsx:27]: please refrain from committing unnecessary refactoring changes,
we would like this to be regressed

**Comment by MadhavDhall** [comment]
Hi sir @AyushDharDubey 
I have made changes in code such that there are no conflicts with base branch. The refactoring changes are hence regressed. 
Kinldy review the code for once again.

File: app/(components)/TicketCard.jsx [hunk 2]
```diff
@@ -40,8 +41,9 @@ const TicketCard = ({ ticket }) => {
         <div className="flex-grow"></div>
         <div className="flex mt-2">
           <div className="flex flex-col">
-            <p className="text-xs my-1">{createdDateTime}</p>
-            {ticket.status === "done" ? "" : <ProgressDisplay progress={ticket.progress} />}
```
Comment (AyushDharDubey) [inline app/(components)/TicketCard.jsx:44]: please restore this status check logic

**Comment by AyushDharDubey** [comment]
good work @MadhavDhall your fix is working and will be merged shortly

**Pull request merged**